app/models/katello/content_override.rb

```diff
@@ -0,0 +1,36 @@
+module Katello
```

Comment:
Let the bike shedding begin, I kid slightly. I feel like there are _more_ files that aren't active record in `app/lib` than in the models directory. If not `app/lib` then perhaps put them in a sub-directory of models to prevent confusion. I know I _should_ check the class inheritance when looking at a file inside the `models` directory, but given how convention over configuration based Rails is, my tendency is going to be to think anything in there is tied to AR.